upb/collections/map.h

```diff
@@ -120,16 +120,18 @@ UPB_API void upb_Map_SetEntryValue(upb_Map* map, size_t iter,
  */
 
 // Advances to the next entry. Returns false if no more entries are present.
-bool upb_MapIterator_Next(const upb_Map* map, size_t* iter);
+UPB_API_INLINE bool upb_MapIterator_Next(const upb_Map* map, size_t* iter);
```

Comment:
`UPB_API_INLINE` is intended for inline functions. For non-inline functions, we should use `UPB_API`.

Reply:
Done